modules/serverless-kafka-broker-tls-default-config.adoc

```diff
@@ -0,0 +1,89 @@
+// Module is included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-kafka.adoc
+
+[id="serverless-kafka-broker-tls-default-config_{context}"]
+= Configuring TLS authentication for the Kafka Broker
+
+.Prerequisites
+
+* A Kafka cluster CA certificate as a `.pem` file.
+* A Kafka cluster client certificate and key as `.pem` files.
+
+.Procedure
+
+. Create the certificate files as secrets in the `knative-eventing` namespace:
++
+[source,terminal]
+----
+$ oc create secret --namespace knative-eventing generic <kafka_auth_secret> \
+  --from-literal=protocol=SSL \
+  --from-file=ca.crt=caroot.pem \
+  --from-file=user.crt=certificate.pem \
+  --from-file=user.key=key.pem
+----
++
+[IMPORTANT]
+====
+Use the key names `ca.crt`, `user.crt`, and `user.key`. Do not change them.
+====
+
+. Start editing the `KnativeKafka` custom resource:
++
+[source,terminal]
+----
+$ oc edit knativekafka
+----
+
+. Reference your secret and the namespace of the secret:
++
+[source,yaml]
+----
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  namespace: knative-eventing
+  name: knative-kafka
+spec:
+  broker:
+    enabled: true
+    defaultConfig:
+      bootstrapServers: <bootstrap_servers>
+      numPartitions: <num_partitions>
+      replicationFactor: <replication_factor>
+      authSecretName: <kafka_auth_secret>
+  channel:
+    enabled: false
+  source:
+    enabled: false
+----
++
+[NOTE]
+====
+Make sure to specify the matching port in the bootstrap server.
+====
++
+For example:
++
+[source,yaml]
+----
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  namespace: knative-eventing
+  name: knative-kafka
+spec:
+  broker:
+    enabled: true
+    defaultConfig:
+      bootstrapServers: eventing-kafka-bootstrap.kafka.svc:9093
+      authSecretName: tls-config
+  channel:
+    enabled: false
+  source:
+    enabled: false
+----
+
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.5/html-single/using_amq_streams_on_rhel/index#assembly-kafka-encryption-and-authentication-str[TLS and SASL on Kafka]
```

Comment:
same comments as above re prerequisite info, formatting objects like `ConfigMap` and duplicating links, this would be better as a one time link in the assembly to reduce maintenance later.

Reply:
Same as in `modules/serverless-kafka-tls.adoc` file, as stated above